assets/js/base/components/price-slider/index.js

```diff
@@ -0,0 +1,405 @@
+/**
+ * External dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+import { Fragment, useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { useDebounce } from '@woocommerce/base-hooks';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { constrainRangeSliderValues, formatCurrencyForInput } from './utils';
+
+const PriceSlider = ( {
+	initialMin,
+	initialMax,
+	min,
+	max,
```

Comment:
I think it might be more descriptive to label these variables with a `constraint` suffix.  So `minConstraint` `maxConstraint`,